public/controllers/register-agent/containers/steps/steps.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { Fragment, useState } from 'react';
+import {
+  EuiSteps,
+  EuiText,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCallOut,
+  EuiPopover,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { InputForm } from '../../../../components/common/form';
+import './steps.scss';
+import {
+  REGISTER_AGENT_DATA_STEP_THREE,
+  REGISTER_AGENT_DATA_STEP_TWO,
+} from '../../utils/register-agent-data';
+import { webDocumentationLink } from '../../../../../common/services/web_documentation';
+import { PLUGIN_VERSION_SHORT } from '../../../../../common/constants';
+
+const popoverServerAddress = (
+  <span>
+    Learn about{' '}
+    <a
+      href={webDocumentationLink(
+        'user-manual/reference/ossec-conf/client.html#groups',
+        PLUGIN_VERSION_SHORT,
+      )}
+      target='_blank'
+      rel='noopener noreferrer'
+    >
+      Server address.
+    </a>
+  </span>
+);
+
+const popoverAgentName = (
+  <span>
+    Learn about{' '}
+    <a
```

Comment:
**suggestion**: use `EuiLink`

Reply:
Resolved in commit [b2c04c2](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/b2c04c26921790fffdc8971eb3596c75acc2c82d)